web/themes/joinup/joinup_theme.theme

```diff
@@ -992,3 +1022,16 @@ function joinup_theme_preprocess_table__file_formatter_table(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Add theme suggestions and classes in search form.
+ */
+function joinup_theme_form_views_exposed_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (strpos($form['#id'], 'views-exposed-form-search-page-') !== FALSE) {
+    $form['keys']['#attributes']['data-twig-suggestion'] = 'search_bar';
```

Comment:
Please move this to another array key that starts with #. Mangling with the #attributes means that they will be printed in the HTML, which in this case is not wanted. 
`$form['keys']['#extra_suggestion'] = 'search_bar';`
And then you can check that this "property" is set on the render array on the template suggestion hook.